.github/workflows/live_tests.yaml

```diff
@@ -1,36 +1,52 @@
 name: Tests against a Live Instance
 
 on:
-  # allow manual runs
+  # Allow manual runs
   workflow_dispatch:
 
-  # run every day
+  # Run every day
   schedule:
-    - cron: "0 0 * * *" 
+    - cron: "0 0 * * *"
 
 jobs:
-  test-against-live-instance:
+  test-live-instance:
     runs-on: ubuntu-latest
     env:
+      # Secrets for API and organization
+      REFRESH_TOKEN: ${{ secrets.DEEP_ORIGIN_REFRESH_TOKEN }}
+      ACCESS_TOKEN: ${{ secrets.DEEP_ORIGIN_ACCESS_TOKEN }}
```

Comment:
it is important that these are not prepended with DEEP_ORIGIN

Reply:
Perhaps leave a comment why, haha.